os/kernel/init/os_bringup.c

```diff
@@ -276,6 +276,10 @@ static inline void os_do_appstart(void)
 	}
 #endif
 
+#ifdef CONFIG_KERNEL_TEST_DRV
+	test_drv_register();
+#endif
+
```

Comment:
Hi, 
test_drv_register() is already being called in os_start.c.
Please remove it from this file.

Reply:
okay thanks. i remove it